### PR TITLE
add: component for US01 .jsx .css and data.json for theme

### PR DESCRIPTION
**Pull request by Lambseb**

### CODE REVIEW

**Review by Chikoulat — APPROVED** [review]

File: frontend/src/components/ButtonsByTheme.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import data from "../data/data.json";
+import "./ButtonsByTheme.css";
+
+function ButtonByTheme(setQuery) {
+  const [themButton, setthemButton] = useState([]);
+  useEffect(() => {
```
Comment (ioayoub) [inline frontend/src/components/ButtonsByTheme.jsx:8]: useEffect is useless here, you can initialize your themeButton with data

**Review by ioayoub — CHANGES_REQUESTED** [review]
Be careful with css files naming, they must be in lowercase.

File: frontend/src/components/ButtonsByTheme.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            >
+              {r.description.toUpperCase()}
+            </button>
+          ) : null
+        )}
+    </div>
+  );
+}
+ButtonByTheme.proTypes = {
+  setQuery: PropTypes.string.isRequired,
```
Comment (ioayoub) [inline frontend/src/components/ButtonsByTheme.jsx:25]: setQuery is a function not a string

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Comment by Lambseb** [comment]
all are fixed.

**Review by ioayoub — APPROVED** [review]

**Comment by gaelkat** [comment]
À

**Review by StalkerVince — APPROVED** [review]

**Pull request merged**